docker-compose.yml

```diff
@@ -115,6 +121,22 @@ services:
     networks:
       - isolation
 
+  gateway:
+    depends_on:
+      firezone:
+        condition: 'service_healthy'
+    image: firezone-gateway:latest
+    privileged: true
+    volumes:
+      - ./priv/gateway-conf/:/app/gateway-conf/
+    sysctls:
+      - net.ipv6.conf.all.disable_ipv6=0
+      - net.ipv4.conf.all.src_valid_mark=1
+    networks:
+      - app
+      - isolation
+    command: /bin/sh -c "gateway --conf-dir ./gateway-conf register http://firezone:13000/auth/gateway/token/kYl25nYBkLF6yCljeu1ccw== && gateway --conf-dir ./gateway-conf start ws://firezone:13000/gateway/websocket"
```

Comment:
All the tokens should be in ENV variables, basically, here we see that we need two: `CONTROL_PLANE_ENDPOINT` and `CONTROL_PANEL_AUTH_TOKEN_SECRET`.

Also, I think we don't have a `register` endpoint anymore. Is this line outdated? Or gateway was not updated yet?

Reply:
Moving all the tokens to env variables in #1311 where I also removed the registration logic